packages/devtools_app/test/network/network_request_inspector_test.dart

```diff
@@ -0,0 +1,78 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:devtools_app/src/config_specific/ide_theme/ide_theme.dart';
+import 'package:devtools_app/src/http/http_request_data.dart';
+import 'package:devtools_app/src/screens/network/network_controller.dart';
+import 'package:devtools_app/src/screens/network/network_request_inspector.dart';
+import 'package:devtools_app/src/service/service_manager.dart';
+import 'package:devtools_app/src/shared/common_widgets.dart';
+import 'package:devtools_app/src/shared/globals.dart';
+import 'package:devtools_test/devtools_test.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:vm_service/vm_service.dart';
+
+import '../test_utils/test_utils.dart';
+import 'utils/network_test_utils.dart';
+
+void main() {
+  group('NetworkRequestInspector', () {
+    late NetworkController controller;
+    late SocketProfile socketProfile;
+    late HttpProfile httpProfile;
+    late FakeServiceManager fakeServiceManager;
+    late String _clipboardContents;
```

Comment:
Nit: if we're always setting this to empty string in `setUp` anyway, we may as well initialize it here too and remove the `late`. It's not super important in this context, but using `late` does introduce additional initialization checks at each access, so if there's a simple way around using `late` you should try and avoid it.